src/components/TodoList/TodoList.tsx

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
 
-export const TodoList: React.FC = () => (
+type Props = {
+  todos: Todo[];
+  onTodoSelected: (_: Todo) => void;
```

Comment:
`_` is used when you need to skip the argument and use the next one. In this case, it should be a proper name for args